mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -79,6 +86,82 @@ bool is_same_socket(sockaddr_storage &r1, sockaddr_storage &r2)
 namespace mavlink_vehicles
 {
 
+namespace math
+{
+
+inline double rad2deg(double x)
+{
+    return (180.0 / M_PI) * x;
+}
+
+inline double deg2rad(double x)
+{
+    return (M_PI / 180.0) * x;
+}
+
+local_pos global_to_local_ned(global_pos_int point, global_pos_int reference)
+{
+    // Scaling factor to convert from 1e-7 degrees to meters at equator
+    // Given by: (M_PI/180)*EARTH_RADIUS_AT_EQUATOR
+    const double location_scaling_factor = 0.011131884502145034f;
```

Comment:
I don't think it is necessary to have this constant into the defaults namespace for some reasons:
1. It will not be used anywhere else.
2. The 'defaults' namespace is reserved only for those constants that might be overridden by the programmer (through a parameter or hard-code). This is not the case with location_scaling_factor, since the functions that use it will not work properly work if its value changes.
3. Also note that the global2local's location_scaling_factor is different from the one in local2global (one is the inverse of the other).